awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -512,7 +512,7 @@ function JobTemplateForm({
                         &nbsp;
                         <FieldTooltip
                           content={i18n._(
-                            t`Enable webhook for this workflow job template.`
+                            t`Enable webhook for this job template.`
```

Comment:
I think we should dynamically change this string based on the template type to be `Enable webhook for this workflow job template` or `Enable webhook for this job template`